tests/foreman/cli/test_oscap.py

```diff
@@ -57,16 +57,26 @@ def test_positive_list_default_content_with_admin(self, module_target_sat):
             1. Login to shell from admin account.
             2. Execute the scap-content command with list as sub-command.
 
-        :expectedresults: Default scap-content are listed.
+        :expectedresults: Default scap-content are listed. They should already be populated on a fresh Satellite instance from Broker.
 
         :BZ: 1749692
+        :verifies: SAT-30938
```

Comment:
surprised that autochecks are not angry about  missing space betw docstring items

Reply:
Fixed so it doesn't make you angry :)